src/components/RiskComponents.vue

```diff
@@ -14,15 +14,39 @@
         <RiskComponent :activity="activity" type="ventilation"></RiskComponent>
       </v-col>
     </v-row>
+    <v-row no-gutters v-if="!onMediumandUp">
+      <v-col cols="12" sm-and-down>
+        <v-expansion-panels flat>
+          <RiskComponentDropdown
+            :activity="activity"
+            type="crowding"
+          ></RiskComponentDropdown>
+          <RiskComponentDropdown
+            :activity="activity"
+            type="droplets"
+          ></RiskComponentDropdown>
+          <RiskComponentDropdown
+            :activity="activity"
+            type="exposureTime"
+          ></RiskComponentDropdown>
+          <RiskComponentDropdown
+            :activity="activity"
+            type="ventilation"
+          ></RiskComponentDropdown>
+        </v-expansion-panels>
+      </v-col>
```

Comment:
you could use v-for here instead of repeating component
you could compute necessary props in this component and use v-bind here to prevent having to pass the whole activity object to the child component — this would also prevent having repeating code between RiskComponent/RiskComponentDropdown

Reply:
Waiting on Brock's PR to merge before doing this.